autofs-5.0.6 - fix rpc build error

From: Leonardo Chiquitto <leonardo.lists@gmail.com>

The compile error looks like a problem in the libtirpc interface code rewrite
(commit f8ea2a5762 - autofs-5.0.6 - fix ipv6 rpc calls). The following patch
should fix it.
---

 CHANGELOG      |    1 +
 lib/rpc_subs.c |    2 +-
 2 files changed, 2 insertions(+), 1 deletions(-)


diff --git a/CHANGELOG b/CHANGELOG
index 8dec17f..11d2cbf 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -20,6 +20,7 @@
 - add piddir to configure.
 - add systemd unit support.
 - remove empty command line arguments (passed by systemd).
+- fix rpc build error.
 
 28/06/2011 autofs-5.0.6
 -----------------------
diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index ba679a5..4082fee 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -154,7 +154,7 @@ static CLIENT *rpc_do_create_client(struct sockaddr *addr, struct conn_info *inf
 {
 	CLIENT *client = NULL;
 	struct sockaddr_in in4_laddr;
-	struct sockaddr_in in4_raddr;
+	struct sockaddr_in *in4_raddr;
 	int type, proto;
 	socklen_t slen;
 
